### ISSUE TITLE
[Bug]: mybatis + MO：execute prepare is not successful

### ISSUE THREAD

**Issue by goodMan-code** [matrixorigin/matrixone#6491]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

commit fcda32551ba7076fa24b8156b5d374da253220c1
launch method : ./mo-service -launch ./etc/launch-tae-CN-tae-DN/launch.toml &> mo.log &
mysql -h 127.0.0.1 -P 6001 -udump -p111

The problem is : 
use postman send GET request to select some data,  this select statement is combined with prepare, It result is NULL,
However this select can be execute successfully on MySQL.

Execute Steps : 
1. download this .zip 
2. [demo.zip](https://github.com/matrixorigin/matrixone/files/9985809/demo.zip)
3. launch MO
4. create database test;
5. create table t(
	id int,
	username int,
	password varchar(50),
	address varchar(50)
);
insert into t values (1, 2, 'b', 'c');
insert into t values (2, 2, 'b', 'c');
insert into t values (3, 2, 'b', 'c');
insert into t values (4, 2, 'b', 'c');
6. use postman to send a GET request with URL : localhost:8080/test/prepare

**Comment by YANGGMM** [comment]
still working on it

**Comment by goodMan-code** [comment]
This bug has been fixed, on commit : 873bc541f9ccb4fbb948688a6f6d91ce2c6939c7,  Recheck it, done.
